lib/window.ts

```diff
@@ -8,13 +8,50 @@ export default function createWindow(currentWindow: Window, channel: Channel): W
   const autoUpdateHeight = () => {
     self.updateHeight()
   }
-  const observer = new MutationObserver(autoUpdateHeight)
+
+  const heightObserverCallback = (mutations: Array<MutationRecord>) => {
+    checkAbsolutePositionedElems(mutations)
+    autoUpdateHeight()
+  }
+
+  const observer = new MutationObserver(heightObserverCallback)
   let oldHeight: number
   let isAutoResizing = false
+  const absolutePositionedElems: Set<Node> = new Set()
 
   const self = { startAutoResizer, stopAutoResizer, updateHeight }
   return self
 
+  function checkAbsolutePositionedElems(mutations: Array<MutationRecord>) {
+    mutations.forEach((mutation) => {
+      if (mutation.type === 'attributes') {
+        if (mutation.target.nodeType === 1) {
```

Comment:
nit: Just being explicit

```suggestion
        if (mutation.target.nodeType === Node.NODE_ELEMENT) {
```